src/cript/api/utils/helper_functions.py

```diff
@@ -41,3 +41,30 @@ def _get_node_type_from_json(node_json: Union[Dict, str]) -> str:
     # if invalid then raise error
     else:
         raise CRIPTJsonNodeError(node_list=node_type_list, json_str=str(node_json))
+
+
+def get_node_type_snake_case(node_type: Union[Any, str]) -> str:
```

Comment:
This is a duplication of  `BaseNode.node_type_snake_case`.

Reply:
they are different because that has access to the node class and is just taking it and converting it to string, allows the user to specify either a class if they have it or a string if they need to and this takes both and gives back the same str answer regardless.